File: LCM and HCF/hcf.cpp
#include<iostream>
using namespace std;
int main()
{
	int n,n1,i;
	cout<<"Enter a no. \n";
	cin>>n>>n1;
	if(n>n1)
	{
		for(int i=n1;i>=0;i++)
		{
			if(n%i==0 && n1%i==0)
			{
				cout<<"HCF= "<<i;
				break;	
			}
		}
	}
	else
	{
		for(i=n;i>=0;i++)
		{
			if(n%i==0 && n1%i==0)
			{
				cout<<"HCF= "<<i;
				break;	
			}
		}
	}
	return 0;
}

File: rivu jackson/HCF.java
import java.util.*;
class HCF
{
    int findHcf(int a,int b)
    {
        int h=1;
        for(int i=1;i<a*b;i++){
            if(a%i==0 && b%i==0)
                h=i;
        }
        return h;
    }
    
    public static void main()
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter 1st number");
        int a=sc.nextInt();
        System.out.println("Enter 2nd number");
        int b=sc.nextInt();
        HCF ob=new HCF();
        int hc=ob.findHcf(a,b);
        System.out.println("HCF of "+a+" and "+b+" is "+hc);
    }
}